tests/integration/examples/test_driver_introduction_example.py

```diff
@@ -105,8 +124,9 @@ def test_driver_introduction_example(uri, auth):
     try:
         s = StringIO()
         with redirect_stdout(s):
+            App.enable_log(logging.INFO, sys.stdout)
```

Comment:
```suggestion
            App.enable_log(logging.DEBUG, sys.stdout)
```

I'm unsure if this should be `DEBUG` or `INFO`. Under normal circumstances, there is not much output (if any at all) to be observed on info level. So for illustrative purposes, debug might be the level of choice. On the flip side, I don't want to encourage people to enable debug logging just for the fun of it. It's noisy and might even be misleading if you don't know what you're looking at.

Reply:
I was also in doubt about the level, I'm setting it as INFO since the DEBUG could be really overwhelming.